python/Ganga/Utility/logging/__init__.py

```diff
@@ -80,7 +59,7 @@ def _get_handlers():
 private_logger = None  # private logger of this module
 
 # main logger corresponds to the root of the hierarchy
-main_logger = _get_logging().getLogger()
+main_logger = logging.getLogger()
```

Comment:
A module will not be imported multiple times as it is cached in `sys.modules`. Unless we have a very messed up `PYTHONPATH` we should be safe.